tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/worker/WorkerSyncSendTest.java

```diff
@@ -199,6 +200,21 @@ public void testFailureForReceiveWithError() {
         Assert.assertTrue(((BBoolean) returns[0]).booleanValue());
     }
 
+    @Test(dataProvider = "FunctionList")
+    public void testSimpleSyncSendFunctions(String funcName) {
+        BRunUtil.invoke(result, funcName);
+    }
+
+    @DataProvider(name = "FunctionList")
```

Comment:
Shall we call this something like
```suggestion
    @DataProvider(name = "syncSendFunctionList")
```
or
```suggestion
    @DataProvider(name = "syncSendFunctions")
```